.travis.yml

```diff
@@ -671,6 +671,18 @@ script:
     - if [ "${ALPAKA_CI_SANITIZERS}" == *"MSan"* ]; then export CMAKE_CXX_FLAGS="${CMAKE_CXX_FLAGS} -fsanitize=memory -fsanitize-memory-track-origins"; fi
 
     #-------------------------------------------------------------------------------
+    # gpu architecture
+    - if [ "${ALPAKA_ACC_GPU_CUDA_ENABLE}" == "ON" ]
+      ;then
+          if [ "${ALPAKA_CUDA_COMPILER}" == "clang" ]
+          ;then
+              export ALPAKA_CUDA_ARCH="20"
```

Comment:
This can be removed if [this](https://github.com/ComputationalRadiationPhysics/alpaka/pull/363/files#r131593736) limitation is removed.